packages/website/app/routes/index.tsx

```diff
@@ -5,33 +5,36 @@ import { useEffect, useRef } from 'react'
 import { ClientOnly } from 'remix-utils'
 
 interface DemoAsset {
-	common: TextFileAssetTask
-	vert: TextFileAssetTask
-	frag: TextFileAssetTask
-	music: BinaryFileAssetTask
+	common: TextFileAssetTask;
+	vert: TextFileAssetTask;
+	frag: TextFileAssetTask;
+	music: BinaryFileAssetTask;
 }
 
 interface DemoProps {
-	antialias?: EngineOptions['antialias']
-	adaptToDeviceRatio?: EngineOptions['adaptToDeviceRatio']
-	engineOptions?: EngineOptions
-	sceneOptions?: SceneOptions
+	antialias?: EngineOptions['antialias'];
+	adaptToDeviceRatio?: EngineOptions['adaptToDeviceRatio'];
+	engineOptions?: EngineOptions;
+	sceneOptions?: SceneOptions;
```

Comment:
Semicolon unnecessary, if your linter reports errors, tell me. I'll try to modify eslint settings.